components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -223,6 +235,35 @@ class SettingsPage extends React.Component<Props, State> {
     return null
   }
 
+  renderPromos = () => {
+    const { currentCountryCode, ui } = this.props.rewardsData
+    const { promosDismissed } = ui
+
+    return (
+      <>
+        {activePromos.map((key: PromoType) => {
+          if (promosDismissed && promosDismissed[key]) {
+            return null
+          }
+
+          const promo = getPromo(key, getLocale) as Promo
+
+          if (!promo.supportedLocales.includes(currentCountryCode)) {
```

Comment:
we should here check if `promo.supportedLocales` is not empty. This way promos with empty array can target the whole world.

Reply:
done